ui/src/views/Roles/ViewRole/index.jsx

```diff
@@ -78,6 +84,18 @@ export default class ViewRole extends Component {
     }
   };
 
+  handleActionDialogClose = () => {
+    this.setState({
+      dialogOpen: false,
+      dialogError: null,
```

Comment:
`dialogError` is not defined in this file.

Reply:
It is also defined in line 29.